tests/payload_tools/fitness.py

```
@@ -0,0 +1,49 @@
+def build_create_workout_payload(
+    name: str = "Bicep Chiller",
+    description: str = "A very intense bicep workout",
+    instructions: str = "Lift the dumbbells",
+    workout_type: str = "rep",
+    equipment: list = []
```

Ruff (0.15.10)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)